src/components/dashboard/Claim.js

```diff
@@ -14,23 +14,53 @@ import logger from '../../lib/logger/pino-logger'
 type ClaimProps = DashboardProps & {
   store: Store
 }
+
+type ClaimState = {
+  loading: boolean,
+  nextClaim: string,
+  claimedToday: any
+}
+
 const log = logger.child({ from: 'Claim' })
 
-class Claim extends Component<ClaimProps, {}> {
+class Claim extends Component<ClaimProps, ClaimState> {
   state = {
-    loading: false
+    loading: false,
+    nextClaim: '23:59:59',
+    claimedToday: {
+      people: '',
+      amount: ''
+    }
   }
+
+  interval = null
+
   goodWalletWrapped = wrapper(goodWallet, this.props.store)
+
   async componentDidMount() {
     //if we returned from facerecoginition then the isValid param would be set
     //this happens only on first claim
+    const { entitlement } = this.props.store.get('account')
```

Comment:
The comment below belongs to the line beneath

Reply:
You're right